person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  # getter and setter in one
+  attr_accessor :name
+  attr_accessor :age
+
+  # getter
+  attr_reader :id
+
+  def of_age?
+    @age >= 18
+  end
+  private :is_of_age?
+
+  def can_use_services?
+    @parent_permission || is_of_age?
+  end
```

Comment:
- [ ] Here line 18 & 21 , used `is_of_age?` but defined method is `of_age?`, kindly correct it.
- [ ] [OPTIONAL] there is also good way to use the private method like this 

```ruby
  def can_use_services? 
    of_age?() || @parent_permission
  end
  private 
  
  def of_age?
    @age >= 18
  end

```
when used private like this all method written below `private` is private method, and above one are public.